.github/workflows/main.yml

```
@@ -66,8 +66,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.10", "3.11", "3.12-dev"]
-
+        python-version: ['3.10', '3.11', '3.12', '3.13-dev']
```

```suggestion
        python-version: ["3.10", "3.11", "3.12", "3.13-dev"]
```
Just keeping quote style consistent